src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -818,6 +819,20 @@ public String getVisibilityDropdownLabel(int qnNumber) {
                                                          + " .visibility-options-dropdown button")).getText().trim();
     }
 
+    public boolean isStaticQnNumberVisible(int qnNumber) {
+        WebElement discardChangesButton =
+                browser.driver.findElement(
+                    By.id(Const.ParamsNames.FEEDBACK_QUESTION_NUMBER_STATIC + "-" + qnNumber));
+        return discardChangesButton.isDisplayed();
+    }
+
+    public boolean isMenuQnNumberVisible(int qnNumber) {
+        WebElement discardChangesButton =
+                browser.driver.findElement(
+                        By.id(Const.ParamsNames.FEEDBACK_QUESTION_NUMBER + "-" + qnNumber));
```

Comment:
I think line 831 and 832 can be combined.  If not combine line 830,831 and 832 into 2 lines.